### PR TITLE
Correct shadow root identifier per WebDriver spec

### PR DESCRIPTION
**Pull request by mathiasbynens**
https://w3c.github.io/webdriver/#dfn-shadow-root-identifier

### CODE REVIEW

File: webdriver/tests/get_element_shadow_root/user_prompts.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def check_user_prompt_closed_without_exception(dialog_type, retval):
         response = get_shadow_root(session, element.id)
         value = assert_success(response)
         assert isinstance(value, dict)
-        assert "shadow-075b-4da1-b6ba-e579c2d3230a" in dict
+        assert "shadow-6066-11e4-a52e-4f735466cecf" in dict
```
Comment (whimboo) [inline webdriver/tests/get_element_shadow_root/user_prompts.py:26]: I feel this specific check is a bit out of place in this file. Would you mind moving it over to `get.py/test_get_shadow_root`?

**Review by whimboo — APPROVED** [review]
Good catch. Seems like it was missed when taking the frame class as template.

File: webdriver/tests/get_element_shadow_root/user_prompts.py (hunk 1 above)
Comment (mathiasbynens) [reply webdriver/tests/get_element_shadow_root/user_prompts.py]: Done; PTAL. Is this what you had in mind?

Comment (whimboo) [reply webdriver/tests/get_element_shadow_root/user_prompts.py]: Yes, thanks.

**Pull request merged**